src/applications/gi/components/profile/BenefitsForm.jsx

```diff
@@ -51,157 +52,169 @@ export class BenefitsForm extends React.Component {
-          alt="Completed an enlistment of:"
-          visible={this.props.giBillChapter === '30'}
-          onChange={this.props.eligibilityChange}
-          onFocus={this.props.handleInputFocus}
-        />
-        <Dropdown
-          label="Are you eligible for the Post-9/11 GI Bill?"
-          name="eligForPostGiBill"
-          options={[
-            { value: 'yes', label: 'Yes' },
-            { value: 'no', label: 'No' },
-          ]}
-          value={this.props.eligForPostGiBill}
-          alt="Are you eligible for the Post-9/11 GI Bill?"
-          visible={this.props.giBillChapter === '31'}
-          onChange={this.props.eligibilityChange}
-          onFocus={this.props.handleInputFocus}
-        />
-        <Dropdown
-          label="How many dependents do you have?"
-          name="numberOfDependents"
-          options={[
-            { value: '0', label: '0 Dependents' },
-            { value: '1', label: '1 Dependent' },
-            { value: '2', label: '2 Dependents' },
-            { value: '3', label: '3 Dependents' },
-            { value: '4', label: '4 Dependents' },
-            { value: '5', label: '5 Dependents' },
-          ]}
-          value={this.props.numberOfDependents}
-          alt="How many dependents do you have?"
-          visible={
-            this.props.giBillChapter === '31' &&
-            this.props.eligForPostGiBill === 'no'
+        <ExpandingGroup open={this.props.militaryStatus === 'spouse'}>
+          <Dropdown
+            label="What's your military status?"
+            name="militaryStatus"
+            options={[
+              { value: 'veteran', label: 'Veteran' },
+              { value: 'active duty', label: 'Active Duty' },
+              {
+                value: 'national guard / reserves',
+                label: 'National Guard / Reserves',
+              },
+              { value: 'spouse', label: 'Spouse' },
+              { value: 'child', label: 'Child' },
+            ]}
+            value={this.props.militaryStatus}
+            alt="What's your military status?"
+            visible
+            onChange={this.props.eligibilityChange}
+            onFocus={this.props.handleInputFocus}
+          />
+          <Dropdown
+            label="Is your spouse on active duty?"
+            name="spouseActiveDuty"
+            options={[
+              { value: 'yes', label: 'Yes' },
+              { value: 'no', label: 'No' },
+            ]}
+            value={this.props.spouseActiveDuty}
+            alt="Is your spouse on active duty?"
+            visible={this.props.militaryStatus === 'spouse'}
+            onChange={this.props.eligibilityChange}
+            onFocus={this.props.handleInputFocus}
+          />
+        </ExpandingGroup>
+        <ExpandingGroup
+          open={
+            this.props.giBillChapter === '31' ||
+            this.props.giBillChapter === '33' ||
+            this.props.giBillChapter === '30'
```

Comment:
conditions for only the warning or info box should not trigger blue bar

Reply:
if there are multiple conditionally displayed questions for a parent question:  they are wrapped in a `div` and the `ExpandingGroup` property `open` is set to be an `or` of all the possible display conditions while letting each individual conditionally displayed questions handle whether is displayed or rendered